telegram/dcs/dc_list.go

```diff
@@ -0,0 +1,14 @@
+package dcs
+
+import "github.com/gotd/td/tg"
+
+// DCList is a list of Telegram DC addresses and domains.
+type DCList struct {
```

Comment:
Probably just `dcs.List` is better here?

Reply:
Done